test/cmd.js

```diff
@@ -895,6 +895,60 @@ describe('express(1)', function () {
         .expect(404, /<h1>Not Found<\/h1>/, done)
       })
     })
+
+    describe('--none', function () {
```

Comment:
Looks like the test is misnamed; the args in here are `--view=none`, not `--none`.